src/resonanceReconstruction/rmatrix/legacy/resolved/SpinGroup/MultiLevelBreitWigner/src/evaluate.hpp

```diff
@@ -0,0 +1,40 @@
+/**
+ *  @brief Evaluate the cross sections at the given energy using MLBW
+ *
+ *  @param[in] energy       the incident energy
+ *  @param[in,out] result   a map containing the accumulated cross sections
+ */
+void evaluate( const Energy& energy,
+               std::map< ReactionID, CrossSection >& result ) {
+
+  // data we need: k, P, phi, rho, g_J
+  const auto channel = this->incidentChannel();
+  const auto waveNumber = channel.waveNumber( energy );
+  const auto spinFactor = channel.statisticalSpinFactor();
+
+  // the pi / k2 factor
+  const CrossSection factor = pi / ( waveNumber * waveNumber ) * spinFactor;
```

Comment:
No biggie, but you could put "factor" way down below, just before the one place it's used.

Reply:
Yeah, I have to do a follow up on this work anyway (to include the header guard work on ENDFtk, etc.) and optimise the calculations so I'll do that later.